src/uikit.cs

```diff
@@ -1688,6 +1764,51 @@ interface UIAccessibilityContainer {
 		[iOS (8,0)]
 		[Export ("setAccessibilityElements:")]
 		void SetAccessibilityElements ([NullAllowed] NSObject elements);
+
+		[iOS (11,0), TV (11,0)]
+		[Export ("accessibilityContainerType", ArgumentSemantic.Assign)]
+		UIAccessibilityContainerType AccessibilityContainerType { get; set; }
+	}
+
+	interface IUIAccessibilityContainerDataTableCell {}
+
+	[iOS (11,0), TV (11,0)]
+	[Protocol, Model]
```

Comment:
not clear if a `[Model]` is needed, is this type user created ?

Reply:
Yea it's not clear from the doc nor the headers, let's un-model it until we're proven it's required.